python/doc/examples/surrogate_modeling/linear_regression/plot_sir.py

```
@@ -0,0 +1,146 @@
+"""
```

Ruff (0.15.15)

[warning] The file is executable but no shebang is present

(EXE002)